RGO.App.Tests/Controllers/HRIS/EmployeeRoleManagerControllerUnitTests.cs

```diff
@@ -455,12 +455,13 @@ public async Task GetAllRolesReturnsOk()
 
         var roles = new List<RoleDto>
         {
-            new(1, "Super Admin"),
-            new(2, "Admin")
+            new RoleDto{ Description = "Super Admin" },
+            new RoleDto{Id = 1, Description = "Admin"},
```

Comment:
this id needs to be 2
```suggestion
            new RoleDto{Id = 2, Description = "Admin"},
```